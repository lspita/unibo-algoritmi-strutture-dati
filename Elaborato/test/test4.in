158
48
50
50
-15 17 12 -6 -1 20 3 -14 7 16 16 15 4 12 -1 -7 -19 7 -12 14 17 8 17 12 -11 -11 -17 -9 6 13 9 11 9 2 -13 -11 1 10 -5 -13 7 -7 -18 -10 -16 3 6 -13 -9 14
-17 -12 1 20 -19 12 -12 -13 -17 16 20 14 8 10 -5 15 19 17 7 -5 -15 -7 8 -13 6 14 13 -9 -17 -17 -14 -14 -7 9 -15 -5 0 -5 4 5 -10 5 -2 20 -6 16 15 -6 12 3
11 17 16 1 -14 3 -4 -20 16 -1 -17 -19 5 -1 10 12 16 -8 -14 -1 17 -2 -17 -3 -2 -1 -8 -6 15 -15 19 7 -17 16 -13 -11 -1 4 -9 -6 -18 -3 -2 10 16 -13 1 11 1 8
-9 -2 -13 -4 15 7 17 -14 -20 13 -9 -20 -1 -4 17 7 8 -5 -8 19 11 -4 -5 -12 -15 -10 -2 9 -18 19 -2 -7 -4 5 11 -8 -6 7 0 -4 -1 13 -3 19 -11 -7 7 17 8 -20
15 0 -4 -8 9 4 2 -12 -6 7 -14 12 -19 5 -2 13 19 12 -1 -1 -12 -20 -9 5 -20 2 18 9 0 7 9 -5 -14 -14 7 17 12 -10 -16 -15 17 -7 19 19 18 -3 13 18 -12 -7
-4 -2 -7 10 -18 -5 -7 -19 -17 14 8 14 -10 15 -18 17 13 16 -12 18 -17 -13 -10 -19 -15 -11 0 18 -14 11 -10 5 -12 6 -6 -9 -20 7 -6 -14 2 -18 -19 12 19 -17 10 11 -19 19
10 -16 -15 20 -13 -8 -12 8 -9 -3 -2 3 -19 6 -10 -3 19 -7 -15 -7 -1 8 -5 2 -1 -5 8 -11 -13 9 -11 18 13 -4 17 20 8 -13 7 -20 6 -14 5 7 13 16 -17 11 -12 -12
-15 9 16 0 -10 -3 17 18 6 -16 6 15 -19 0 -10 20 2 20 -12 -10 -19 14 -3 6 2 -11 1 5 -19 11 16 -13 -1 13 9 10 -11 6 9 17 12 -5 13 13 17 -16 14 -2 -17 -18
9 -16 16 -13 13 -2 -4 -7 -16 -1 -16 20 8 5 15 -3 -6 -17 -16 -15 -20 16 0 13 10 -2 18 3 17 -18 5 7 -11 3 14 -19 -20 -9 -4 -14 10 2 -12 -2 -13 -18 17 3 -15 -20
8 -13 18 -11 -20 7 8 20 -9 6 -17 17 -8 -6 -1 -13 -5 1 -2 14 9 -11 -5 17 7 4 19 5 -13 -16 7 15 -7 6 -14 -5 -8 14 -6 6 -1 0 2 13 14 -18 20 -9 5 -3
-16 -7 6 1 -8 -5 -15 -10 -19 -6 -4 9 -12 10 -6 -4 -16 9 -11 1 -6 10 -20 18 2 16 -19 1 -14 6 20 -8 -20 6 14 -8 -18 -20 3 -17 -6 -20 12 4 10 -12 -20 15 17 -9
-5 -8 -19 -3 -11 3 -8 -10 -14 -1 18 -15 13 18 13 8 -8 17 8 15 -20 -17 17 14 10 7 -17 12 3 20 -18 20 -9 -17 -4 2 9 10 12 15 -10 9 20 -17 7 15 11 19 11 1
15 11 4 11 7 -5 -1 10 -14 -19 12 -11 -20 -18 -6 -2 5 -18 -12 -2 17 -2 -13 19 1 16 13 -6 16 3 15 12 -6 0 4 -18 16 -17 12 -19 -16 3 -8 -13 8 7 5 -6 9 13
12 8 -7 -20 6 14 16 -2 -11 13 -20 -17 4 -4 4 -12 0 1 -7 -8 4 -1 17 17 6 4 5 -7 20 -5 -13 12 -18 0 12 8 -4 10 -15 5 2 -12 9 -12 6 -6 -2 -14 17 11
0 0 -11 -4 -2 16 -18 -18 -10 -19 -2 -3 13 0 18 7 -10 -7 -2 -3 18 -19 7 8 -11 -7 -17 9 1 20 -1 -20 2 9 -2 20 6 0 -16 16 2 4 -8 -4 -15 -9 -16 -5 6 4
15 4 6 1 -7 17 16 -3 6 -4 -2 -16 -4 20 15 17 -19 0 -4 -15 -5 20 9 10 -2 14 -20 2 -9 7 -14 -15 -8 14 6 7 10 2 -15 -5 18 4 2 -4 5 -4 -8 6 17 10
13 -7 10 1 -16 -11 -3 -13 13 8 16 19 13 -11 14 0 16 4 -19 -17 -20 -19 9 2 -1 -7 -20 13 -19 19 2 14 -7 -7 -3 -2 2 16 7 -5 3 2 -5 -3 11 -9 17 8 15 -20
11 17 -19 -1 1 0 13 1 -6 16 1 16 9 14 -9 -13 -7 15 -18 20 -9 7 1 9 -15 -7 20 -18 -19 16 -16 12 12 -15 -9 -6 7 -15 17 -20 -20 -3 18 11 -9 -12 19 6 -15 -18
6 -4 12 -12 -16 17 2 -17 19 3 19 -18 -4 13 -11 7 -14 18 12 -17 20 14 0 20 5 13 -13 5 -1 -8 7 -16 11 -2 -6 15 -3 16 19 -5 0 17 0 17 11 11 5 18 11 -3
-20 10 -10 0 9 17 -5 19 1 14 -8 -10 20 -16 10 -7 20 -14 -9 18 4 11 16 -15 9 7 16 -6 6 8 11 6 0 -18 -13 -10 -19 2 -10 2 -3 5 12 -2 9 4 12 8 10 -16
7 -7 15 5 -2 4 -9 -4 18 17 -15 10 4 5 14 11 16 15 -5 -15 -1 14 10 -8 -8 -1 16 -17 -12 8 -13 16 -18 -17 0 1 9 13 19 8 11 -17 -3 -6 11 -7 -14 6 -11 3
13 8 -3 5 -18 9 -17 20 14 -9 7 -20 -12 10 -15 8 -8 15 0 -8 2 -10 -3 -19 4 -13 -4 13 15 5 -5 8 -5 12 -6 -1 2 -3 -2 -5 10 -15 -5 19 15 3 8 -12 -3 -12
0 -20 0 17 -17 -14 -14 -1 19 -20 -16 -5 10 1 -13 -17 20 9 2 -3 -14 -6 4 3 -6 0 -15 -19 8 4 -11 -11 5 12 -12 8 18 -6 -13 16 -6 -7 -10 -14 16 -1 -11 17 -10 14
-5 -4 -13 20 19 1 -1 -14 2 -11 10 14 0 -6 5 8 -16 2 -19 -7 -1 -2 6 11 4 3 -8 15 -1 2 8 14 20 17 15 20 -3 -4 -15 -19 5 16 17 -16 -9 3 15 -5 -14 16
8 7 -7 -5 -3 17 18 9 14 -2 -8 3 -8 -9 0 -12 -9 19 4 -2 20 -10 -7 18 -3 5 2 -9 20 10 -14 10 -4 0 -16 14 -2 -16 4 -7 2 17 17 16 -11 -4 -16 2 -6 8
-20 -4 -20 -5 -5 -3 -19 18 8 -17 7 14 13 -17 -5 19 19 16 -18 4 -10 7 2 -14 2 12 4 8 -5 19 -2 -3 -6 -2 15 9 15 16 8 4 19 -5 -3 13 0 15 12 0 12 16
-17 -17 2 6 -10 -14 -1 14 14 16 14 -9 -7 -13 9 -13 19 5 -16 8 -12 -15 -16 8 18 4 4 11 -17 -3 7 -11 0 -12 15 10 -4 -5 6 -10 -10 -1 1 5 9 -8 14 9 19 19
-4 -13 -17 1 17 -18 -14 0 14 -10 17 2 1 -4 12 -5 -12 -12 12 14 -2 -19 -8 -20 9 -18 -6 2 11 -7 2 -11 2 7 10 -2 10 16 18 5 -13 16 -14 8 -7 18 -16 1 -15 18
-4 5 -19 11 6 10 13 -19 -7 4 -6 16 13 16 2 4 -7 -7 1 -8 20 11 -13 -15 0 1 -15 7 -17 -8 4 0 20 5 -10 5 -6 -18 6 8 8 -18 5 2 18 -14 -15 -7 2 9
5 1 -1 15 9 19 -3 14 5 0 -13 -9 20 -14 16 -9 13 -8 -4 1 20 -17 3 6 8 2 12 15 17 -5 3 4 17 -17 -2 5 -16 15 0 11 -4 7 -19 -2 14 18 9 6 -9 -16
-14 -8 -10 11 18 18 -7 11 12 -11 -14 -3 13 -18 0 -8 9 4 -12 -12 -5 5 15 -4 -18 10 -5 11 -4 8 18 4 20 -13 -5 17 -16 8 8 18 19 14 -6 12 18 15 -15 8 0 -6
16 17 -20 12 -5 -18 2 10 15 18 -2 14 1 -1 -20 16 -4 -14 5 -15 -16 4 -20 0 -3 20 -4 2 7 17 18 2 13 18 -7 -13 -19 17 20 19 14 -1 12 -3 18 15 -6 -5 -20 20
0 -14 5 0 7 -19 -1 -18 5 -15 19 5 9 14 4 -16 -18 8 -20 -19 8 17 2 1 -7 0 -5 7 15 -2 8 -3 4 -8 17 -10 -5 -2 -5 -20 5 -5 5 -4 -12 -10 0 -7 18 2
-6 5 -2 19 -14 13 -2 1 2 -6 -20 -11 11 5 4 10 17 19 -11 -9 -20 14 9 7 -9 19 18 11 -9 15 -5 8 2 13 6 8 8 -15 -10 -11 1 -9 0 -6 16 -17 -17 14 -16 -6
-13 -16 -11 16 12 1 14 11 -7 -13 7 8 15 -12 3 2 16 -10 7 -12 -1 -13 -1 -20 3 -4 -16 6 -11 -12 -19 -4 -6 -9 -7 -13 14 -11 18 -14 -4 7 15 -8 15 -3 14 13 9 2
-20 -11 11 -1 -11 -6 17 -7 -19 -15 3 -17 4 20 -4 19 -14 -11 -13 -14 -5 3 13 -9 17 7 10 10 1 -2 -7 1 7 -16 -19 17 0 18 -9 1 -15 17 4 11 16 -19 9 -19 -10 19
-13 6 3 20 17 0 9 9 -9 -11 -14 5 13 16 11 16 12 -10 16 -16 11 -20 -20 -3 13 16 -2 3 18 9 3 -14 -4 -14 -12 -6 6 19 -18 19 -11 -9 3 -19 -14 -7 18 20 3 13
-17 -4 15 -16 13 9 -19 -8 -7 19 -18 17 -13 0 -18 -5 15 10 -7 19 8 4 -9 -7 6 -3 8 5 -2 -9 -1 2 9 -5 8 3 -15 9 15 -2 10 20 -6 17 -1 -2 -7 -5 -13 8
-4 18 -9 7 -10 19 -16 20 5 2 -10 -17 -15 -20 -2 15 3 5 4 0 -17 -5 1 -3 -7 20 18 8 -3 -16 -5 13 -17 8 -1 -5 9 5 -5 -7 -11 7 -3 -6 8 17 -11 -8 2 15
14 5 -9 -6 -17 4 -6 -20 -7 11 -13 10 5 -8 -3 -17 8 -15 9 -16 -1 18 11 18 -7 0 16 4 15 -3 -2 8 -17 11 -18 -11 -4 -2 -9 9 -10 0 -2 15 13 16 20 2 -18 8
6 3 7 -2 2 -20 -20 -2 4 17 17 -16 4 -18 15 8 -7 -9 -13 4 20 -1 -16 -2 -7 19 -5 -5 2 -1 -18 -13 -16 12 7 6 12 9 -15 -5 5 -18 1 -9 -14 -4 19 -1 7 -12
-16 8 7 -10 -13 -18 -12 4 -3 10 -17 1 19 -11 -8 8 15 -17 -4 -18 1 -17 -14 -19 -6 -8 -1 -8 13 -13 0 17 16 -11 -13 -18 -9 -3 7 8 -13 12 -12 -13 -20 3 15 17 6 -7
19 -12 -4 -16 -10 10 -2 9 -17 -10 17 6 -11 14 15 -4 18 -15 13 6 15 -19 -3 -16 -11 -1 7 -15 -4 -7 0 -4 1 16 3 13 7 -20 4 11 -7 0 -4 2 -5 -10 20 -8 -3 14
19 -9 16 -3 -5 -14 17 -18 -7 -8 -3 14 10 20 11 -8 13 -2 -5 -4 -12 8 18 6 -9 -6 18 -10 6 -6 -17 4 5 -20 -19 -19 -12 20 -15 2 -7 2 -5 -18 2 -13 -3 -4 5 12
14 15 1 13 0 12 -14 -3 -17 12 13 -12 -2 -3 -9 1 1 -1 2 6 -20 15 -12 -3 17 12 7 -7 -13 -9 -14 -20 -14 7 15 8 0 -20 -15 5 14 18 14 -7 -3 -14 16 18 5 -3
5 8 -9 13 -14 -10 6 13 5 13 -16 12 15 -8 -2 9 20 -20 9 -14 5 5 -14 0 18 3 6 14 2 -7 -10 -14 -20 2 -19 -12 14 7 -19 -2 2 -13 -11 -2 -1 10 -13 0 10 18
7 -4 2 15 19 2 -3 4 -5 -20 17 7 -12 19 -10 -9 -13 -17 -20 -10 2 4 -3 13 -19 18 2 -12 -3 -7 -12 -15 12 12 20 10 -7 -2 -5 10 0 -8 -3 11 -8 9 -19 1 13 -17
11 -4 7 -13 -12 9 -16 13 19 4 -15 -14 9 19 19 11 8 -7 -12 -17 -17 11 -3 2 1 11 -10 2 -8 -16 8 -18 0 -6 -8 11 -18 -4 3 -19 -19 10 -11 13 9 -11 3 -2 3 13
1 8 3 20 -9 -15 11 1 10 -18 8 -3 -14 -13 11 -2 18 16 17 2 19 18 -8 -13 12 -18 -1 -6 0 -19 -11 -17 11 14 -18 -19 20 15 4 9 17 -9 -15 -17 -1 18 3 -2 13 -1
-18 13 -1 -4 -18 -10 -2 1 7 18 4 16 -18 -6 9 -15 -3 10 20 -20 -2 19 -8 5 -17 13 5 6 -8 -1 -15 -6 -8 6 10 -6 16 -11 17 2 -11 0 -1 -9 16 -12 -2 -8 20 0
