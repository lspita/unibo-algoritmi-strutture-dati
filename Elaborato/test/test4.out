0 0
1 0
1 1
1 2
2 2
2 3
3 3
4 3
4 4
4 5
3 5
2 5
2 6
1 6
1 7
0 7
0 8
0 9
0 10
0 11
1 11
1 12
2 12
2 13
2 14
2 15
3 15
3 16
3 17
3 18
4 18
4 19
4 20
5 20
5 21
5 22
6 22
6 23
7 23
8 23
9 23
10 23
11 23
12 23
13 23
14 23
15 23
15 22
16 22
17 22
17 21
18 21
19 21
20 21
20 22
21 22
22 22
23 22
23 23
23 24
23 25
24 25
24 26
25 26
25 27
26 27
27 27
28 27
29 27
30 27
31 27
32 27
33 27
34 27
35 27
35 28
35 29
35 30
35 31
34 31
34 32
34 33
35 33
36 33
36 34
36 35
37 35
38 35
39 35
39 36
39 37
39 38
38 38
38 39
38 40
39 40
39 41
39 42
40 42
40 43
41 43
41 42
42 42
42 43
42 44
42 45
43 45
43 46
43 47
43 48
44 48
45 48
46 48
47 48
48 48
48 49
49 49
-1 -1
263814
